packages/manager/src/features/GlobalNotifications/CreditCardExpiredBanner.tsx

```diff
@@ -0,0 +1,50 @@
+import { Typography } from '@mui/material';
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { Button } from 'src/components/Button/Button';
+import { DismissibleBanner } from 'src/components/DismissibleBanner/DismissibleBanner';
+import { useAllPaymentMethodsQuery } from 'src/queries/account/payment';
+import { isCreditCardExpired } from 'src/utilities/creditCard';
+
+import type { PaymentMethod } from '@linode/api-v4/lib/account/types';
+
+export const CreditCardExpiredBanner = () => {
+  const history = useHistory();
+
+  const { data: paymentMethods } = useAllPaymentMethodsQuery();
+
+  if (!paymentMethods || paymentMethods?.length == 0) {
+    return;
+  }
+
+  const isExpired = paymentMethods.some((paymentMethod: PaymentMethod) => {
+    const ccExpiry =
+      paymentMethod.type === 'credit_card' ? paymentMethod.data.expiry : null;
+    return ccExpiry && isCreditCardExpired(ccExpiry);
+  });
```

Comment:
Optional: This might be a slightly cleaner way to represent this: 

```suggestion
  const isExpired = paymentMethods.some((paymentMethod) => {
    if (paymentMethod.type === 'credit_card' && paymentMethod.data.expiry) {
      return isCreditCardExpired(paymentMethod.data.expiry);
    }
    return false;
  });
```